Libraries/Components/Touchable/TouchableBounce.js

```diff
@@ -175,10 +175,7 @@ class TouchableBounce extends React.Component<Props, State> {
                 !this.props.disabled,
             }
           : {
-              focusable:
-                this.props.focusable !== false &&
-                this.props.onPress !== undefined &&
-                !this.props.disabled,
+              focusable: this.props.focusable !== false && !this.props.disabled,
```

Comment:
We should add a TODO tag if this differs from upstream. Also this seems like a candidate to contribute back upstream. Thoughts @HeyImChris ?

Reply:
These Touchable* sections are all within blocks marked `// [macOS` #656 `...` `// macOS]`

Reply:
I like the idea of checking this in to fb's repo. If it's within the macOS tags, let's make sure there's a TODO with the issue number tracking it. This is a good candidate to try pushing upstream to FB's repo and see what they think about it. It might have a lot of fallout of causing all their clients to need to set the prop so they might decline, but worth trying...